e2etest/pkg/service/tweak.go

```diff
@@ -25,3 +25,9 @@ func DualStack(svc *corev1.Service) {
 func TrafficPolicyCluster(svc *corev1.Service) {
 	svc.Spec.ExternalTrafficPolicy = corev1.ServiceExternalTrafficPolicyTypeCluster
 }
+
+func WithSpecificIP(svc *corev1.Service, ip string) {
```

Comment:
this can't be used to return multiple IPs to test dual-stack

Reply:
yes, changed it to var-args now.